index.html

```diff
@@ -6607,19 +6632,26 @@ <h3>Placeholder</h3>
 
 <p>
     A <a>Thing Model</a> can specify which terms should be used in a TD instance, but their values are unspecific and are first known during TD instantiation. 
-    In such a case the placeholder labeling MAY be used in Thing Model that MUST be substituted with a concrete value when TD instance is created 
-    from the Thing Model. 
+    <span class="rfc2119-assertion" id="tm-placeholder-usecase">
+        In a case where TD instance terms, but not their values, are known in advance, the placeholder labeling MAY be used in a Thing Model.
+    </span>
+    <span class="rfc2119-assertion" id="tm-placeholder-replacement">
+        The placeholder labeling MUST be substituted with a concrete value when TD instance is created from the Thing Model. 
+    </span>
     <span class="rfc2119-assertion" id="tm-placeholder">
         The string-based pattern of the placeholder MUST follow a valid pattern based on the regular expression <code>{{2}[ -~]+}{2}</code> (e.g., <code>{{</code><code>PLACEHOLDER_IDENTIFIER</code><code>}}</code>). 
         The characters between <code>{{</code> and <code>}}</code> are used as identifier name of the placeholder. The identifier name can be used to identify the placeholder for the substitution process.
     </span>
     <span class="rfc2119-assertion" id="tm-placeholder-value">
-        A placeholder can only be applied within the value of the JSON name-value pair and the value has to be a JSON string. 
+        A placeholder MUST be applied within the value of the JSON name-value pair. 
     </span>  
     <span class="rfc2119-assertion" id="tm-placeholder-retyping">
-    In the case that a non string-based value of a JSON name-value pair should have a placeholder, the value must be (temporarily) typed as string. After replacing the placeholder, e.g. when creating a Thing Description instance, the original type can be applied with the 
-    corresponding replaced value.  
+        In the case that a non string-based value of a JSON name-value pair SHOULD have a placeholder, the value MUST be (temporarily) typed as string. 
```

Comment:
problematic, should not have two RFC-2119 keywords in a single assertion.  In this case, however, the SHOULD is a condition, so reword to says something like "If X then Y MUST B"

Reply:
removed should and changed to if